fedoratagger/frontend/templates/master.mak

```diff
@@ -44,5 +44,27 @@
 			<a href="https://github.com/fedora-infra/fedora-tagger/issues">
 			the GitHub issues tracker</a>.
 		</div>
+
+    % if 'FEDMENU_URL' in g.config:
```

Comment:
How does mako react if you try to access an un-defined variable?

In other words, should we check for `FEDMENU_DATA_URL` as well?